<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>851</width>
      <height>591</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="Tab1">
     <attribute name="title">
      <string>Raw Data</string>
     </attribute>
     <widget class="QListWidget" name="CBUListWidget">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>50</y>
        <width>321</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelForCBUList">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Loaded CBU Files</string>
      </property>
     </widget>
     <widget class="QListWidget" name="DuesListWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>311</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelForDuesList">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Loaded Dues Deductions Files</string>
      </property>
     </widget>
     <widget class="QPushButton" name="RemoveFileButton">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove File</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ReconcileEveryThingButton">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>370</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Reconcile Everything</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ReconcileButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>420</y>
        <width>83</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Reconcile</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelDuesFile">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>420</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Nothing Selected</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelCBUFile">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>420</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Nothing Selected</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Annual Reconcilation</string>
     </attribute>
     <widget class="QListWidget" name="SelectedFilesForAnnualRec">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>250</y>
        <width>241</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>0</y>
        <width>636</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>AFT</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>AFT MI</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Salary Cut off</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>1/8 Time</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Member Per Cap</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Fee Payer per Cap</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>1/4 Time</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Pay Level</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Member Per Cap</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>1/2 Time</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Full Time</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Fee Payer per Cap</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="FullTime_CutOff"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="HalfTime_CutOff"/>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="QuarterTime_CutOff"/>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="EighthTime_CutOff"/>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="FullTime_AFTMemPerCap"/>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="HalfTime_AFTMemPerCap"/>
         </item>
         <item row="3" column="2">
          <widget class="QLineEdit" name="QuarterTime_AFTMemPerCap"/>
         </item>
         <item row="4" column="2">
          <widget class="QLineEdit" name="EighthTime_AFTMemPerCap"/>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="FullTime_AFTFeePerCap"/>
         </item>
         <item row="2" column="3">
          <widget class="QLineEdit" name="HalfTime_AFTFeePerCap"/>
         </item>
         <item row="3" column="3">
          <widget class="QLineEdit" name="QuarterTime_AFTFeePerCap"/>
         </item>
         <item row="4" column="3">
          <widget class="QLineEdit" name="EighthTime_AFTFeePerCap"/>
         </item>
         <item row="1" column="4">
          <widget class="QLineEdit" name="FullTime_AFTMIMemPerCap"/>
         </item>
         <item row="2" column="4">
          <widget class="QLineEdit" name="HalfTime_AFTMIMemPerCap"/>
         </item>
         <item row="3" column="4">
          <widget class="QLineEdit" name="QuarterTime_AFTMIMemPerCap"/>
         </item>
         <item row="4" column="4">
          <widget class="QLineEdit" name="EighthTime_AFTMIMemPerCap"/>
         </item>
         <item row="1" column="5">
          <widget class="QLineEdit" name="FullTime_AFTMIFeePerCap"/>
         </item>
         <item row="2" column="5">
          <widget class="QLineEdit" name="HalfTime_AFTMIFeePerCap"/>
         </item>
         <item row="3" column="5">
          <widget class="QLineEdit" name="QuarterTime_AFTMIFeePerCap"/>
         </item>
         <item row="4" column="5">
          <widget class="QLineEdit" name="EighthTime_AFTMIFeePerCap"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="runAnnualRec">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>260</y>
        <width>110</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>166</x>
        <y>220</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Selection</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>220</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Other Selection</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Search</string>
     </attribute>
     <widget class="QTextBrowser" name="searchOutputBox">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>611</width>
        <height>441</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="SearchLine">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>846</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
